client/landing/stepper/declarative-flow/internals/components/stepper-loader/index.tsx

```diff
@@ -1,10 +1,36 @@
+import { ProgressBar } from '@wordpress/components';
 import clsx from 'clsx';
-import WordPressLogo from 'calypso/components/wordpress-logo';
 import './style.scss';
 
-const StepperLoader = () => {
-	/* eslint-disable wpcalypso/jsx-classname-namespace */
-	return <WordPressLogo size={ 72 } className={ clsx( 'wpcom-site__logo', 'stepper-loader' ) } />;
+interface StepperLoaderProps {
+	title?: string;
+	subtitle?: React.ReactNode;
+	progress?: number;
+	className?: string;
+}
+
+const StepperLoader: React.FC< StepperLoaderProps > = ( {
+	title,
+	subtitle,
+	progress = -1,
+	className,
+} ) => {
+	const renderProgressComponent = () => {
```

Comment:
Nit: This function seems useless now right? we can just embed the React element.

Reply:
Addressed 👍